install.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/sh
+# TODO(everyone): Keep this script simple and easily auditable.
+
+set -e
+
+if [ "$(uname -m)" != "x86_64" ]; then
+  echo "Error: Unsupported architecture $(uname -m). Only x64 binaries are available." 1>&2
+  exit 1
+fi
+
+if [ "$OS" = "Windows_NT" ]; then
+  target="windows-386"
+  ext=".zip"
+  if ! command -v unzip >/dev/null; then
+    echo "Error: unzip is required to install coder-cli" 1>&2
+    exit 1
+  fi
+else
+  if ! command -v tar >/dev/null; then
+    echo "Error: tar is required to install coder-cli" 1>&2
+    exit 1
+  fi
+  ext=".tar.gz"
+  case $(uname -s) in
+  Darwin) target="darwin-amd64" ;;
+  *) target="linux-amd64" ;;
+  esac
+fi
+
+if [ $# -eq 0 ]; then
+  coder_asset_path=$(
+    curl -sSf https://github.com/cdr/coder-cli/releases |
+      grep -o "/cdr/coder-cli/releases/download/.*/coder-cli-${target}${ext}" |
+      head -n 1
+  )
+  if [ ! "$coder_asset_path" ]; then
+    echo "Error: Unable to find latest coder-cli release on GitHub." 1>&2
+    exit 1
+  fi
+  cdr_uri="https://github.com${coder_asset_path}"
+else
+  cdr_uri="https://github.com/cdr/coder-cli/releases/download/${1}/coder-cli-${target}${ext}"
+fi
+
+coder_install="${CODER_INSTALL:-$HOME/.coder}"
```

Comment:
This should be `~/.local/lib/coder`.

See https://github.com/cdr/code-server/blob/11b51058355caa60d21f59b09996d127776c8c84/install.sh#L42-L46

Reply:
Is there a specific standard for this that I'm unaware of? I find having these kinds of hidden directories in my home folder convenient. Especially if we add configuration files to this directory down the line.

Reply:
Yes, the XDG standard. It's `~/.local`. And then `~/.config` for the configuration files.

Reply:
So there are zero programs on my machine in `~/.local/lib`. Inclined to ignore the standard.